queries.sql

```diff
@@ -1,3 +1,25 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- Find all animals whose name ends in "mon".
+SELECT * FROM ANIMALS WHERE NAME LIKE '%mon';
+
+-- List the name of all animals born between 2016 and 2019.
+SELECT NAME FROM ANIMALS WHERE DATE_OF_BIRTH >= '2016-1-1' AND DATE_OF_BIRTH <= '2019-12-30';
+
+-- List the name of all animals that are neutered and have less than 3 escape attempts.
+SELECT NAME FROM ANIMALS WHERE NEUTERED IS TRUE AND ESCAPE_ATTEMPTS <= 3;
```

Comment:
- [ ] the escape attempts should be strictly less than 3, kindly consider using the `<`, not the `<=` in this case
```suggestion
-- List the name of all animals that are neutered and have less than 3 escape attempts.
SELECT NAME FROM ANIMALS WHERE NEUTERED IS TRUE AND ESCAPE_ATTEMPTS < 3;
```